packages/open-api-gateway/src/project/spec/open-api-spec-project.ts

```diff
@@ -45,7 +74,7 @@ export class OpenApiSpecProject extends Project {
         "..",
         "..",
         "samples",
-        this.specFileName
+        "spec.yaml"
```

Comment:
shouldn't this be `this.specFileName`?

Reply:
Nope, this is the name of the sample yaml file so it's fixed :)